apps-rendering/src/item.ts

```diff
@@ -367,150 +374,151 @@ const isPicture = hasTag('type/picture');
 
 const fromCapiLiveBlog =
 	(context: Context) =>
-	(
-		request: RenderingRequest,
-		blockId: Option<string>,
-	): LiveBlog | DeadBlog => {
-		const { content } = request;
-		const body = content.blocks?.body ?? [];
-		const pageSize = content.tags.map((c) => c.id).includes('sport/sport')
-			? 30
-			: 10;
-
-		const parsedBlocks = parseLiveBlocks(context)(body, content.tags);
-		const pagedBlocks = getPagedBlocks(pageSize, parsedBlocks, blockId);
-		return {
-			design:
-				content.fields?.liveBloggingNow === true
-					? ArticleDesign.LiveBlog
-					: ArticleDesign.DeadBlog,
-			blocks: parsedBlocks,
-			pagedBlocks,
-			totalBodyBlocks: content.blocks?.totalBodyBlocks ?? body.length,
-			...itemFields(context, request),
+		(
+			request: RenderingRequest,
+			blockId: Option<string>,
+		): LiveBlog | DeadBlog => {
+			const { content } = request;
+			const body = content.blocks?.body ?? [];
+			const pageSize = content.tags.map((c) => c.id).includes('sport/sport')
+				? 30
+				: 10;
+
+			const parsedBlocks = parseLiveBlocks(context)(body, content.tags);
+			const pagedBlocks = getPagedBlocks(pageSize, parsedBlocks, blockId);
+			return {
+				design:
+					content.fields?.liveBloggingNow === true
+						? ArticleDesign.LiveBlog
+						: ArticleDesign.DeadBlog,
+				blocks: parsedBlocks,
+				pagedBlocks,
+				totalBodyBlocks: content.blocks?.totalBodyBlocks ?? body.length,
+				...itemFields(context, request),
+			};
 		};
-	};
```

Comment:
As far as I can tell nothing has changed here? Is this diff all indentation? 🤔

Reply:
Yep all whitespace changes - I've reverted this as requested.